examples/neptune_tracking/pipelines/training_pipeline/training_pipeline.py

```diff
@@ -12,11 +12,19 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
+from zenml.config import DockerSettings
+from zenml.integrations.constants import NEPTUNE, SKLEARN, TENSORFLOW
 from zenml.pipelines import pipeline
 
+docker_settings = DockerSettings(
+    required_integrations=[TENSORFLOW, SKLEARN, NEPTUNE],
```

Comment:
I couldn't see any mentions of sklearn in the example, do you remember why that was necessary?

Also maybe as an additional point, I would also remove neptune from this list of required integrations as ZenML already detects the integration of all stack components and installs them

Reply:
Yes sklearn is not needed, will remove it from list of required integrtions.